trax/models/research/funnel_transformer.py

```diff
@@ -351,6 +362,8 @@ def FunnelTransformer(vocab_size,
                                   dropout_shared_axes, mode, ff_activation)
                     for _ in range(n_decoder_blocks)]
 
+  total_pool_size = pool_size * (len(encoder_segment_lengths) - 1)
```

Comment:
should be `pool_size ** (len(encoder_segment_lengths) - 1)` ?

Reply:
done